app/src/androidTest/java/com/twilio/video/app/e2eTest/LoginTest.kt

```diff
@@ -27,15 +28,24 @@ class LoginTest {
     fun `it_should_login_successfully_with_email_and_then_logout`() {
         val emailCredentials = retrieveEmailCredentials()
         loginWithEmail(emailCredentials)
-        retryViewMatcher { clickSettingsMenuItem() }
+        retryEspressoAction { clickSettingsMenuItem() }
+
+        // Test config change
+        // TODO Replace with scenario.recreate once app uses single activity
+        uiDevice().run {
```

Comment:
I think this should be a separate test as it was before. In general, it's good practice to keep tests single purpose.

Reply:
I agree, especially after looking at this again with fresh eyes 😄